techLogger.js

```diff
@@ -72,47 +74,29 @@ function _makeLogger(prefix) {
             return updated;
         },
 
-        debug: function() {
-            _log("debug", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        debug: (...args) => _log("debug", _prefixLog(args, logPrefix)),
 
-        info: function() {
-            _log("info", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        info: (...args) => _log("info", _prefixLog(args, logPrefix)),
 
-        notice: function() {
-            _log("notice", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        notice: (...args) => _log("notice", _prefixLog(args, logPrefix)),
 
-        warn: function() {
-            _log("warning", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        warn: (...args) => _log("warning", _prefixLog(args, logPrefix)),
 
-        error: function() {
-            _log("error", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        error: (...args) => _log("error", _prefixLog(args, logPrefix)),
 
-        crit: function() {
-            _log("crit", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        crit: (...args) => _log("crit", _prefixLog(args, logPrefix)),
 
-        alert: function() {
-            _log("alert", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
-        },
+        alert: (...args) => _log("alert", _prefixLog(args, logPrefix)),
 
-        emerg: function() {
-            _log("emerg", _prefixLog(Array.prototype.slice.call(arguments), logPrefix));
+        emerg(...args) {
+            return _log("emerg", _prefixLog(args, logPrefix));
         },
 
-        createExpressLoggerStream: function(level) {
-            return {
-                write: function(message /*, encoding */ ) {
-                    _log(level, message);
-                }
-            };
+        createExpressLoggerStream(level) {
+            return { write: (message /*, encoding */ ) => _log(level, message) };
         },
 
-        splash: function(app, configuration) {
+        splash(app, configuration) {
             splash(this, app, configuration);
         }
```

Comment:
Another option for your splash object definition could be to use object short notation and removing `this` parameter to the call.

```
return {
    debug: (...args) => _log("debug", _prefixLog(args, logPrefix)),
    //...
    splash(app, configuration) {
        splash(this, app, configuration);
    }
};
```
could then simply become:
```
return {
    debug: (...args) => _log("debug", _prefixLog(args, logPrefix)),
    //...
    splash
};
```